tests/testthat/test-script.R

```diff
@@ -0,0 +1,61 @@
+
+context("install-github.R script")
+
+test_that("install-github.R script is up to date", {
+  root <- system.file(package = .packageName)
```

Comment:
Maybe we should use `utils::packageName()`? 

> > So, should I use .packageName or utils::packageName() for this?
> 
> You should use utils::packageName.  It is documented; .packageName is not.
> 
> Duncan Murdoch

(https://www.mail-archive.com/r-devel@r-project.org/msg35792.html)

Reply:
OK